src/diffusers/modular_pipelines/modular_pipeline.py

```diff
@@ -322,7 +322,7 @@ class ModularPipelineBlocks(ConfigMixin, PushToHubMixin):
     </Tip>
     """
 
-    config_name = "config.json"
+    config_name = "modular_config.json"
```

Comment:
For consistency with `modular_model_index.json`. Also could be cases where a repo contains a model file and a modular pipeline block to load the model. We can avoid conflicts with the configs this way.